src/invoke.cpp

```diff
@@ -762,11 +762,12 @@ void fcall_value( lk::invoke_t &cxt )
 		{
 			if ( vv->Read( cxt.arg(1), false ) ){
 			    bool trigger = true;
-			    if (cxt.arg_count() == 3 )
+			    if (cxt.arg_count() == 3 ) {
```

Comment:
While I agree that this seems to be what is implied by the indentation, adding this curly brace will fundamentally change the behavior of this function. @sjanzou , can you comment whether this looks like the intended behavior, or whether we just need to change indenting for clarity?

Reply:
@janinefreeman, good catch, the proposed change is incorrect (verified in VS2019) 
![image](https://user-images.githubusercontent.com/6498311/85377304-2392d000-b4f6-11ea-8ec8-21ad5989ef21.png)
and needs to be 
![image](https://user-images.githubusercontent.com/6498311/85377384-3b6a5400-b4f6-11ea-98b3-feb46428bf42.png)


and needs to be: